lookout/style/format/uast_stability_checker.py

```diff
@@ -141,7 +141,6 @@ def _check_file(
             vnode_start_index = start_offset_to_vnodes[start]
             vnode_end_index = end_offset_to_vnodes[end]
 
-            assert vnode_start_index <= vnodes_i < vnode_end_index
```

Comment:
Let's keep it just in case. It is harmless.

Reply:
Or is it triggered because
> also the general invariant "A parent covers its children" is not respected in the js driver

? 

Then we should fix `UastStabilityChecker` because it uses this assumption. 
In such a case, it is another problem for another PR then, I think

Reply:
It is triggered indeed.

Reply:
let's keep an assert because it means that we still should address it properly.
I think I know how.

Reply:
I'd rather fix UastStabilityChecker not to rely on this invariant

Reply:
whatever you decide, I am fine with both options

Reply:
Ok, let's keep it as it is now, I created an issue to do not lose it. https://github.com/src-d/style-analyzer/issues/733